package.json

```diff
@@ -1,8 +1,9 @@
 {
   "scripts": {
-    "test": "xo"
+    "test": "xo && ava"
   },
   "devDependencies": {
+    "ava": "^0.13.0",
```

Comment:
```
"ava": "*",
```

I prefer using `*` in my own things to always be on latest so I can catch error before my users do.